src/ClaimService.Mappers/Db/DbClaimMapper.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using LT.DigitalOffice.ClaimService.Mappers.Db.Intterfaces;
+using LT.DigitalOffice.ClaimService.Models.Db;
+using LT.DigitalOffice.ClaimService.Models.Dto.Enums;
+using LT.DigitalOffice.ClaimService.Models.Dto.Requests;
+
+namespace LT.DigitalOffice.ClaimService.Mappers.Db;
+
+public class DbClaimMapper : IDbClaimMapper
+{
+  public DbClaim Map(CreateClaimRequest request, Guid senderId)
+  {
+    return request is null
+      ? null
+      : new DbClaim
+      {
+        Id = Guid.NewGuid(),
+        Name = request.Name,
+        CategoryId = request.CategoryId,
+        Content = request.Content,
+        Status = ClaimStatus.Created,
+        Urgency = request.Urgency.HasValue ? request.Urgency.Value : ClaimUrgency.Magor,
```

Comment:
Just make Magor default value for Urgency in request

Reply:
in that case I'll lose option to set an Urgency

Reply:
Why? Adding default value means that this value will be used if no value provided

Reply:
ok, I understand